### DISCUSSION TITLE
Older MacBook pro?

### DISCUSSION THREAD

**Discussion by bostongarden** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#471] [Q&A]
Will this work on MacBook Pri (13-inch, Mid 2012) running El Capitan (10.11.6)? If not is there any chance it will be improved to include older models?

**Comment by MatthiasKerbl** [comment]
Hi @bostongarden,

You can update your MacBook to macOS Catalina and run AlDente Classic 2.2. This should work even on your MacBook model. You can download AlDente Classic 2.2 here: https://github.com/davidwernhart/AlDente/releases/download/2.2/AlDente_2.2_Notarized.app.zip

Cheers,
Matthias

**Comment by bostongarden** [comment]
Hello Matthias,

Thanks for your response! I do not see the option to update my MacBook to
OS Catalina in About This Mac/Software Update which takes me to the App
Store updates section (which says no updates available).

Are you sure that macOS Catalina is compatible and if so how would I do the
update?

Best regards,
Dave
WhatsApp/SMS: 781-354-0510

**Comment by MatthiasKerbl** [comment]
Hi @bostongarden,

Your MacBook should be compatible with macOS Catalina(https://support.apple.com/en-us/HT210222).
Here are the instructions from Apple on how to update: https://support.apple.com/en-gb/HT211683

Cheers,
Matthias

**Comment by bostongarden** [comment]
Thank you very much!
Kind regards,
Dave
WhatsApp/SMS: 781-354-0510

**Comment by bostongarden** [comment]
Hi Matthias,
I successfully updated to Catalina and installed AlDente. Thanks for your
support. It has been working properly for a few days now. I set to 80% and
it stops charging there.
I notice that my battery seems to be running a lot hotter than before. Is
this expected or reported by others? Is this normal? Seems like that would
defeat the intended battery life improvement...
Thanks
Dave

**Reply by MatthiasKerbl** [reply]
Hi @bostongarden,

I am glad you got it to work.
Your battery should definitely not run hotter when AlDente is installed. You can check in activity monitor how much CPU resources AlDente uses. Usually, it should be around 1%, if that is the case, AlDente should not be the cause for the hotter battery.

Cheers,
Matthias

**Comment by benfade** [comment]
hello, also trying to install on a mid 2012 with Catalina. But I get an error saying -47 code and there's no app available that can open the file.
![Bildschirmfoto 2023-12-28 um 12 55 52](https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/119120889/d687c31d-685b-4f4d-a424-535cf4242da2)
![Bildschirmfoto 2023-12-28 um 12 58 12](https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/119120889/e92bebfb-db52-42ad-9560-df85ccb93acd)
